redaxo/src/addons/structure/CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+Version 2.9.0 – XX.XX.2020
+--------------------------
+
+### Neu
+
+* Neue Klasse `rex_template_select` für die Template-Auswahl (@DanielWeitenauer)
```

Comment:
Muss jetzt auch ins content plugin

Reply:
Changelogs haben wir nur pro Addon, nicht in deren Plugins.